azurerm/internal/services/network/data_source_loadbalancer.go

```diff
@@ -141,10 +147,14 @@ func dataSourceArmLoadBalancerRead(d *schema.ResourceData, meta interface{}) err
 
 						privateIpAddresses = append(privateIpAddresses, *feipProps.PrivateIPAddress)
 					}
+					if privateIpAddressVersion == "" {
+						privateIpAddressVersion = string(feipProps.PrivateIPAddressVersion)
+					}
 				}
 			}
 
 			d.Set("private_ip_address", privateIpAddress)
+			d.Set("private_ip_address_version", privateIpAddressVersion)
```

Comment:
Setting `private_ip_address_version` at this level causes a crash because it lives under `frontend_ip_configuration`. I believe we can just remove this line